common/src/test/scala/uk/ac/wellcome/models/MergedSierraRecordTest.scala

```diff
@@ -0,0 +1,130 @@
+package uk.ac.wellcome.models
+
+import java.time.Instant
+
+import org.scalatest.{FunSpec, Matchers}
+
+import uk.ac.wellcome.utils.JsonUtil
+
+class MergedSierraRecordTest extends FunSpec with Matchers {
+
+  it("should allow creation of MergedSierraRecord with no data") {
+    MergedSierraRecord(id = "111")
+  }
+
+  it("should merge data from a bibRecord when empty") {
+    val record = sierraBibRecord(id = "222")
+    val originalRecord = MergedSierraRecord(id = "222")
+
+    val newRecord: Option[MergedSierraRecord] =
+      originalRecord.mergeBibRecord(record)
+
+    newRecord.get.bibData.get shouldEqual record
+  }
+
+  it("should only merge bib records with matching ids") {
+    val record = sierraBibRecord(id = "333")
+    val originalRecord = MergedSierraRecord(id = "444")
+
+    val caught = intercept[RuntimeException] {
+      originalRecord.mergeBibRecord(record)
+    }
+    caught.getMessage shouldEqual "Non-matching bib ids 333 != 444"
```

Comment:
it it can throw exceptions, would it be morth making it return a Try or an Either?